packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -1600,4 +1601,19 @@ export class JobService {
     jobEntity.status = JobStatus.COMPLETED;
     await this.jobRepository.updateOne(jobEntity);
   }
+
+  public async isEscrowFunded(
```

Comment:
This function has no tests

Reply:
Unit tests for `isEscrowFunded` were implemented.